Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,27 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        if ($formMapper->hasOpenTab()) {
```

Comment:
In that case, we can't merge any of these tab support PRs:

| Sonata version | Tab used | Result |
| --- | --- | ---
| 2.3.* | no | Weird experience, as extension uses tabs |
| 2.3.* | yes | Error, tab not closed before executing extension |
| 2.4.* | no | Weird experience |
| 2.4.* | yes | The correct experience |

I understand your thoughts, I'm also a big unsure of dropping sonata <2.4 support. However, we have to decide between that and a good experience.